tests/console/yast2_dns_server.pm

```diff
@@ -11,7 +11,7 @@
 # Summary: Ensure that all combinations of running/stopped and active/inactive can be set for yast2 dns-server
 # Maintainer: jeriveramoya <jeriveramoya@suse.com>
 
-use base qw(console_yasttest y2logsstep);
+use base qw(console_yasttest);
```

Comment:
We also can get rid of `qw` here as have single value.

Reply:
Done at 10eea2bec